recipes/limix/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "limix" %}
+{% set version = "0.8.3" %}
+{% set sha256 = "5c025ed38b84bfd52373d9a2f4758903903be50f88da5d553dfd27c40791000d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --compatible --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
```

Comment:
Yep. When you have `https://github.com/PMBio/limix/blob/master/setup.py#L142` in your code you need `numpy x.x`, at both `build` and `run` sections, to baked the `numpy` build string in the package name. That ensures the right `numpy` version will be used with the package.